runtime/connectors/connectors.go

```diff
@@ -80,11 +80,12 @@ func (ps PropertySchema) ValidateType(val any) bool {
 // and (in the future) secrets configured by the user.
 type Env struct {
 	RepoDriver string
-	RepoDSN    string
+	RepoRoot   string // should be absolute
```

Comment:
`should be absolute` - do we have a check for this ? I see we are using `.` in the tests, will this resolve to an absolute path ?

Reply:
Actually it was more of an FYI for path access checks to correctly work. I now added an explicit check to convert `RepoRoot` into an absolute path where this is being used and removed the comment.